lib/vet360/models/address.rb

```diff
@@ -160,7 +160,7 @@ def self.build_from(body)
           address_pou: body['address_pou'],
           address_type: body['address_type'],
```

Comment:
I think we'll need an upcase of some sort here to make sure we always caps the type like the constants above declare

Reply:
@rlbaker I'm interested in why this should happen here in the de-serializer? Isn't this what our validation rules are for? It'd be easy to `.upcase` these here... but that feels like introducing a random sprinkle of sanitization that we don't do anywhere else. Should we also be capitalizing `country_name`? Or `upcase`'ing phone types?

Reply:
@rlbaker Done